doc/reference/provided_metrics.md

```diff
@@ -154,6 +158,19 @@ The following internal metrics are provided:
   - Number of active warnings
 ```
 
+(api-rates-metrics)=
+## API rates metrics
+
+The API rates metrics include `lxd_api_requests_completed_total` and `lxd_api_requests_ongoing`. These metrics can be consumed by an observability tool deployed externally (for example, the [Canonical Observability Stack](https://charmhub.io/topics/canonical-observability-stack) or another third-party tool) to help identify failures or overload on a LXD server. You can set thresholds on the observability tools for these metrics' values to trigger alarms and take programmatic actions.
+
+These metrics consider all endpoints in the [LXD REST API](../api), with the exception of the `/` endpoint. Requests using an invalid URL are also counted. Requests against the metrics server are also counted. Both introduced metrics include a label `entity_type` based on the main entity type that the endpoint is operating on.
```

Comment:
>  Requests against the metrics server are also counted.

What entity type is used for this?

Reply:
Since the type is determined by the endpoint, it would be`typeServer`. Same as the regular rest server